compendium/ConfigurationAdmin/test/TestConfigurationAdminImpl.cpp

```diff
@@ -41,6 +41,40 @@ MATCHER_P(AnyMapEquals, value, "")
   cppmicroservices::any_value_to_string(valueString, value);
   return (argString.str() == valueString.str());
 }
+/* This is a ConfigurationListener that blocks on a mutex. It is used by the TestConfigurationAdminImpl
+ * VerifyNoUpdateDeadlock test to recreate the blocking condition that caused a deadlock for the 
+ * Update, UpdateIfDifferent and Remove Configuration methods
+ */
+struct BlockingConfigurationListener final
+  : public cppmicroservices::service::cm::ConfigurationListener
+{
+
+  BlockingConfigurationListener() = default;
+  BlockingConfigurationListener(const BlockingConfigurationListener&) = delete;
+  BlockingConfigurationListener(BlockingConfigurationListener&&) = delete;
+  BlockingConfigurationListener& operator=(
+    const BlockingConfigurationListener&) = delete;
+  BlockingConfigurationListener& operator=(BlockingConfigurationListener&&) =
+    delete;
+  ~BlockingConfigurationListener() = default;
+
+  /*
+   * configurationEvent is the method called by Configuration Admin whenever a 
+   * a configuraton object is updated or removed.
+   *
+   */
+  void configurationEvent(
```

Comment:
I would make the first letter of the function name ('c') capital to match the formatting we use throughout the codebase

Reply:
configurationEvent is a method defined in the OSGI spec